features/list-elements.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
-name: <li>
-description: The `<li>` element TODO.
+name: List elements.
```

Comment:
The name of this is challenging in a similar way to `<h1>` through `<h6>`. I think that "&lt;ol> and &lt;ul>" similar to https://github.com/web-platform-dx/web-features/pull/1374 would be OK here, since `<li>` only makes sense in the context of another element. The description should mention all elements and the word "list" so that it can be found in search. The name of the file would be `ol-ul.yml`.